### ISSUE TITLE
Screen reader reads each part of Mail Row instead of interacting with whole row as one

### ISSUE THREAD

**Issue by wrdhub** [tutao/tutanota#9403]
When using a VoiceOver (iOS) or TalkBack (android), the screen reader reads through each part of a mail list item (sender, time, subject, icons) individually instead of going over each item in the list. This means you have to swipe multiple times to get to the next item and is pretty annoying. You should be able to go through the list items a whole pieces.

**Comment by charlag** [comment]
We had some success fixing it by setting `role` explicitly (also when it should be redundant), adding `tabindex`, hiding all children of the row and adding `aria-label`.

We run into some issues where some elements were not selected correctly. It looked like `touch` and `click` events are sometimes dispatched inconsistently. We suspect that it has something to do with invalid size of focus ring. We narrowed it down to transform on any of the items inside the row (which we have for animating multiselection transitions). We are unsure what to do about it yet.
